projects/user_publication/scopus.py

```diff
@@ -93,8 +106,16 @@ def pub_import(task, dry_run=True):
                 link=link,
                 status=Publication.STATUS_SUBMITTED,
             )
-            publications.append((PublicationSource.SCOPUS, pub_model))
+            publications.append(
+                RawPublicationSource(
+                    pub_model=pub_model,
+                    source_id=raw_pub.eid,
+                    source_name=PublicationSource.SCOPUS,
+                )
+            )
         except Exception as e:
-            logger.error(f"Error processing publication {raw_pub}: {e}")
+            # TODO  we keep hitting this
```

Comment:
I used to see a bunch of errors with scopus here without much info, I don't see them anymore, so I expect it was mostly an issue with API rate limiting.